phi3-finetune/environment/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2004-cu118-py38-torch220
+
+RUN pip uninstall onnxruntime-training -y && \
```

Comment:
ACPT has a 1.18.0 onnxruntime-training package. Doesn't it support Phi-3 model? Why is the package re-installed?

Reply:
It should work. Let me give that a try